packs/familiar-abilities/plant-form.json

```diff
@@ -11,12 +11,12 @@
         },
         "category": "familiar",
         "description": {
-            "value": "<p>Your plant familiar can change shape as a single action, transforming into a Tiny plant of a type roughly similar to the familiar's nature. This otherwise uses the effects of <em>tree shape</em>. You must have a familiar with the plant trait, such as a leshy, to select this ability.</p>"
+            "value": "<p>Your plant familiar can change shape as a single action, transforming into a Tiny plant of a type roughly similar to the familiar's nature. This otherwise uses the effects of <em>@UUID[Compendium.pf2e.spells-srd.Item.One with Plants]</em>. You must have a familiar with the plant trait to select this ability.</p>"
```

Comment:
```suggestion
            "value": "<p>Your plant familiar can change shape as a single action, transforming into a Tiny plant of a type roughly similar to the familiar's nature. This otherwise uses the effects of @UUID[Compendium.pf2e.spells-srd.Item.One with Plants]. You must have a familiar with the plant trait to select this ability.</p>"
```